esmvaltool/recipes/cmorizers/recipe_era5.yml

```
@@ -0,0 +1,343 @@
+# ESMValTool
+# recipe_era5.yml
+---
+documentation:
+  description: CMORize ERA5 data
+
+  authors:
+    - andela_bouwe
+    - kalverla_peter
+
+  references:
+    - acknow_project
+
+  projects:
+    - ewatercycle
+
+datasets:
+  - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3, start_year: 1993, end_year: 1993}
```

```suggestion
  - {dataset: ERA5, project: native6, type: reanaly, version: '1', tier: 3, start_year: 1990, end_year: 1990}
```
For consistency with the `ERA5` check in the `recipe_check_obs.yml` (otherview I need to download 1993 for all variables).